examples/E2EBert/news_data_handler.py

```diff
@@ -189,39 +146,28 @@ def create_data_loader(self, df, tokenizer, max_len, batch_size):
             ds, batch_size=batch_size, num_workers=3
         )
 
-    def train_dataloader(self):
-        """
-        :return: output - Train data loader for the given input
-        """
-        self.train_data_loader = self.create_data_loader(
-            self.df_train, self.tokenizer, self.MAX_LEN, 16
-        )
-        return self.train_data_loader
-
-    def val_dataloader(self):
-        """
-        :return: output - Validation data loader for the given input
-        """
-        self.val_data_loader = self.create_data_loader(
-            self.df_val, self.tokenizer, self.MAX_LEN, 16
-        )
-        return self.val_data_loader
-
-    def test_dataloader(self):
-        """
-        :return: output - Test data loader for the given input
-        """
-        self.test_data_loader = self.create_data_loader(
-            self.df_test, self.tokenizer, self.MAX_LEN, 16
-        )
-        return self.test_data_loader
-
-    def get_batch_data(self):
+    @staticmethod
+    def get_batch_data():
```

Comment:
@chauhang As of now @KasirajanA has used the data loader and batched (Captum Batches) the data.
Should the batching of data be done on the training side ? if so, user has to do the batching.

Or we should the data batching on the Captum Visualizer part.
The problem in batching the data on captum side is : it is tough to generalize the batching concept across different models.
May be, we can divide into sub categories like `image_classifier`, `text_classifer` etc. User can either use prediefined batching logic or write of his own (providing both the options).

Please let us know your thoughts

Reply:
@shrinath-suresh, @chauhang as suggested I stored the dataset (input tensor and target class) into state dict itself and generate batch in captum_visualizer script.